specification/cognitiveservices/data-plane/AutoSuggest/stable/v1.0/AutoSuggest.json

```diff
@@ -0,0 +1,632 @@
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "ResponseFormat",
+            "in": "query",
+            "description": "The media type to use for the response. The following are the possible case-insensitive values: JSON, JSONLD. The default is JSON. If you specify JSONLD, the response body includes JSON-LD objects that contain the search results.",
+            "required": false,
+            "type": "array",
+            "items": {
+              "type": "string",
+              "enum": [
+                "Json",
+                "JsonLd"
+              ],
+              "x-ms-enum": {
+                "name": "ResponseFormat",
+                "modelAsString": true
+              }
+            },
+            "collectionFormat": "csv"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success.",
+            "schema": {
+              "$ref": "#/definitions/Suggestions"
+            }
+          },
+          "default": {
+            "description": "An error has occurred. Check the response type and/or status code for more details.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Successful query": {
+            "$ref": "./examples/SuccessfulQueryRequest.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Suggestions": {
+      "allOf": [
+        {
+          "$ref": "#/definitions/SearchResultsAnswer"
+        }
+      ],
+      "type": "object",
+      "required": [
+        "suggestionGroups"
+      ],
+      "properties": {
+        "suggestionGroups": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SuggestionsSuggestionGroup"
+          }
+        }
+      }
+    },
+    "SearchResultsAnswer": {
+      "description": "Defines a search result answer.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Answer"
+        }
+      ],
+      "type": "object",
+      "properties": {
+        "queryContext": {
+          "$ref": "#/definitions/QueryContext",
+          "readOnly": true
+        }
+      }
+    },
+    "SuggestionsSuggestionGroup": {
```

Comment:
According to https://github.com/Azure/azure-rest-api-specs/pull/3180/files#diff-670740650e285f2c72a7d3ced0e1e73cR18, this either needs to have `_type` property or extend from `ResponseBase`.

Reply:
update spec, thank you